jax/scipy/special.py

```diff
@@ -32,6 +32,12 @@ def gammaln(x):
   return lax.lgamma(x)
 
 
+@_wraps(osp_special.betaln)
+def betaln(x, y):
+  x, y = _promote_args_like(osp_special.betaln, x, y)
+  return lax.sub(lax.add(lax.lgamma(x), lax.lgamma(y)), lax.lgamma(x + y))
```

Comment:
nit: I would prefer either *always* using infix operators like `+`/`-` or *never* using them in favor of `add`/`sub` functions. Right now you're using a strange mix...

Reply:
Done.